app/models/user.rb

```diff
@@ -83,6 +83,9 @@ def self.to_csv
   scope :unconfirmed, -> { where(confirmed_at: nil) }
   scope :locked_out, -> { where.not(locked_at: nil) }
 
+  # @return [Integer] Number of users who have added at least one note, given the note body is not empty
```

Comment:
The scope should be returning a collection of filtered records, which we can then count when reporting.